velox/exec/PlanNodeStats.h

```diff
@@ -104,7 +104,7 @@ struct PlanNodeStats {
 
   std::string toString(bool includeInputStats = false) const;
 
-  bool isMultiOperatorNode() const {
+  bool isMultiOperatorTypeNode() const {
```

Comment:
nit: Next time consider extracting this change into a separate PR to keep the currently PR small and focused.

Reply:
will do, thanks for calling that out.